akka-docs/src/main/paradox/stream/operators/Source-or-Flow/flatMapConcat.md

```diff
@@ -4,18 +4,32 @@ Transform each input element into a `Source` whose elements are then flattened i
 
 @ref[Nesting and flattening operators](../index.md#nesting-and-flattening-operators)
 
-@@@div { .group-scala }
 
 ## Signature
 
-@@signature [Flow.scala](/akka-stream/src/main/scala/akka/stream/scaladsl/Flow.scala) { #flatMapConcat }
+@apidoc[Flow.flatMapConcat](Flow) { scala="#flatMapConcat[T,M](f:Out=%3akka.stream.Graph[akka.stream.SourceShape[T],M]):FlowOps.this.Repr[T]" java="#flatMapConcat(akka.japi.function.Function)" } 
 
-@@@
+See also: @ref:[flatMapMerge](flatMapMerge.md)
```

Comment:
I think for the others we have put see-also links in the description section rather than under signature